### ISSUE TITLE
Uno.XamlBehaviors: Does not work for WebAssembly.

### ISSUE THREAD

**Issue by HavenDV** [unoplatform/uno#5285]
This code works correctly for a Uno Uwp project, but not for WebAssembly.
```xaml
<Grid>
    <Grid.ColumnDefinitions>
        <ColumnDefinition/>
        <ColumnDefinition/>
    </Grid.ColumnDefinitions>
    <SplitView 
        x:Name="MySplitView"
        PaneBackground="Gray"
        DisplayMode="CompactOverlay"
        IsPaneOpen="True"
        CompactPaneLength="50"
        OpenPaneLength="280"
        >
        <SplitView.Pane>
            <Grid>
                <Button Content="close inside"
                    HorizontalAlignment="Center"
                    VerticalAlignment="Center"
                    >
                    <Interactivity:Interaction.Behaviors>
                        <Interactions:EventTriggerBehavior EventName="Click" >
                            <Interactions:GoToStateAction 
                                TargetObject="{Binding ElementName=MySplitView}"
                                StateName="Closed"
                                />
                        </Interactions:EventTriggerBehavior>
                    </Interactivity:Interaction.Behaviors>
                </Button>
            </Grid>
        </SplitView.Pane>

        <SplitView.Content>
            <Grid>
                <Button 
                    Content="open inside"
                    HorizontalAlignment="Center"
                    VerticalAlignment="Center"
                    >
                    <Interactivity:Interaction.Behaviors>
                        <Interactions:EventTriggerBehavior EventName="Click">
                            <Interactions:GoToStateAction 
                                TargetObject="{Binding ElementName=MySplitView}"
                                StateName="OpenOverlayLeft"
                                />
                        </Interactions:EventTriggerBehavior>
                    </Interactivity:Interaction.Behaviors>
                </Button>
            </Grid>
        </SplitView.Content>
    </SplitView>

    <Grid 
        Grid.Column="1"
        Background="Aquamarine"
        >
        <Grid.ColumnDefinitions>
            <ColumnDefinition />
            <ColumnDefinition />
        </Grid.ColumnDefinitions>
        <Button 
            Content="open outside"
            HorizontalAlignment="Center"
            VerticalAlignment="Center"
            >         
            <Interactivity:Interaction.Behaviors>
                <Interactions:EventTriggerBehavior EventName="Click">
                    <Interactions:GoToStateAction 
                        TargetObject="{Binding ElementName=MySplitView}"
                        StateName="OpenOverlayLeft"
                        />
                </Interactions:EventTriggerBehavior>
            </Interactivity:Interaction.Behaviors>
        </Button>

        <Button 
            Content="close outside"
            Grid.Column="1"
            HorizontalAlignment="Center"
            VerticalAlignment="Center"
            >
            <Interactivity:Interaction.Behaviors>
                <Interactions:EventTriggerBehavior EventName="Click" >
                    <Interactions:GoToStateAction 
                        TargetObject="{Binding ElementName=MySplitView}"
                        StateName="Closed"
                        />
                </Interactions:EventTriggerBehavior>
            </Interactivity:Interaction.Behaviors>
        </Button>
    </Grid>
</Grid>
```

Expected behavior:
![image](https://i.stack.imgur.com/YGf17.gif)

Google Chrome Log:
![image](https://user-images.githubusercontent.com/3002068/108587459-5d76e780-739f-11eb-906c-2a44dcedf94f.png)

**Comment by jeromelaban** [comment]
Thanks for the report. This is likely to be a Linker error. Make sure to [read this section](https://platform.uno/docs/articles/get-started-wizard.html#runtime-error-no-parameterless-constructor-defined-for-xxxx) for more details.

**Comment by HavenDV** [comment]
This happens in Debug mode when WasmShellILLinkerEnabled is false.

**Comment by jeromelaban** [comment]
Thanks. This may be caused by something else. Could you create a sample that reproduces the issue and attach it to this issue ?

**Comment by HavenDV** [comment]
Here's the project file. The xaml code is self-contained and does not reference any external elements.
```xml
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net5.0</TargetFramework>
    <ApplicationIcon>icon.ico</ApplicationIcon>
    <NoWarn>($NoWarn);NU1701;CS8669;Uno0001</NoWarn>
  </PropertyGroup>

  <PropertyGroup Label="WASM">
    <WasmHead>true</WasmHead>
    <WasmShellEnableLongPathSupport>false</WasmShellEnableLongPathSupport>
    <DefineConstants>$(DefineConstants);__WASM__</DefineConstants>
    <MonoRuntimeDebuggerEnabled Condition=" '$(Configuration)' == 'Debug' ">true</MonoRuntimeDebuggerEnabled>
    <WasmShellILLinkerEnabled Condition=" '$(Configuration)' == 'Debug' ">false</WasmShellILLinkerEnabled>
  </PropertyGroup>

  <ItemGroup>
    <LinkerDescriptor Include="LinkerConfig.xml" />
    <Content Include="LinkerConfig.xml" />
    <EmbeddedResource Include="WasmCSS\Fonts.css" />
    <EmbeddedResource Include="WasmScripts\AppManifest.js" />
  </ItemGroup>

  <ItemGroup>
    <PackageReference Include="ReactiveUI.Uno" Version="13.1.1" />
    <PackageReference Include="Uno.Microsoft.Toolkit.Uwp.UI.Controls" Version="6.1.0-build.207.gcac670e618" />
    <PackageReference Include="Uno.Microsoft.Toolkit.Uwp.UI.Controls.DataGrid" Version="6.1.0-build.207.gcac670e618" />
    <PackageReference Include="Uno.Microsoft.Xaml.Behaviors.Uwp.Managed" Version="2.0.2-uno.81" />
    <PackageReference Include="Uno.UI.WebAssembly" Version="3.5.0-dev.482" />
    <PackageReference Include="Uno.Wasm.Bootstrap" Version="2.0.0-dev.167" />
    <PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="2.0.0-dev.167" PrivateAssets="all" />
    <PackageReference Include="Microsoft.Windows.Compatibility" Version="5.0.1" />
  </ItemGroup>

</Project>
```

**Comment by jeromelaban** [comment]
Thanks for the update. We generally request a full reproduction sample, created from a Uno Template for easier troubleshooting. Could you provide this ?

**Comment by HavenDV** [comment]
Fixed in 3.7.
